src/App.jsx

```diff
@@ -1,19 +1,37 @@
-import React, { useState } from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
 import "./App.css";
+import React from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
+
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = React.useState(
```

Comment:
If you want to clean up your code a little more can actually destructure React hooks from your `react` module removing the need to call a hook on the `React` class.

Reply:
Yesterday I had that doubt, thanks for clarifying for me.